content/account_management/faq/is-it-possible-to-have-my-login-belong-to-multiple-datadog-organizations.md

```diff
@@ -3,9 +3,9 @@ title: Is it possible to have my login belong to multiple Datadog organizations?
 kind: faq
 ---
 
-It is! There is the [multi-account feature][1]  to help creating new organizations that is not currently enabled by default so the first thing you'll have to do is send a note to support@datadoghq.com asking them to enable new account creation for your organization.
+Yes. The [multi-account feature][1] enables you to create new organizations. To enable this feature for your organization, email support@datadoghq.com.
```

Comment:
Change `email support@datadoghq.com` to `contact [Datadog support][3]`. Then add the `/help` link to bottom.